frontend/src/service/GraphqlService.re

```diff
@@ -8,19 +8,26 @@ type query('a) = {
 };
 
 let executeQuery = (grapqhQuery: query('a)): PromiseUtils.t('a) =>
-  Fetch.fetchWithInit(
-    Environment.graphqlUrl,
-    Fetch.RequestInit.make(
-      ~method_=Post,
-      ~body=
-        Js.Dict.fromList([("query", Js.Json.string(grapqhQuery##query)), ("variables", grapqhQuery##variables)])
-        |> Js.Json.object_
-        |> Js.Json.stringify
-        |> Fetch.BodyInit.make,
-      ~headers=Fetch.HeadersInit.makeWithArray([|("content-type", "application/json")|]),
-      (),
-    ),
-  )
+  {
+    let (keycloak, _) = Keycloak.useKeycloak();
```

Comment:
https://reactjs.org/docs/hooks-rules.html
Only Call Hooks at the Top Level

The hook mustn't be used here. This is an example of how this can be done https://github.com/panz3r/react-keycloak/blob/master/src/demo/utils/hooks.js, but we have no time so please add `TODO: Hook cannot be used here`.